zbus_macros/src/proxy.rs

```diff
@@ -193,15 +262,51 @@ pub fn create_proxy(
 
     for i in input.items.iter() {
         if let syn::TraitItem::Method(m) = i {
-            let mut attrs = MethodAttributes::parse(&m.attrs)?;
+            let (
+                mut name,
+                signal,
+                _object,
+                _async_object,
+                _blocking_object,
+                _no_reply,
+                _no_autostart,
+                _allow_interactive_auth,
```

Comment:
why initialize all these just to ignore? :thinking:

Reply:
This is a big advantage of keeping commits atomic: it's easy to see if unwanted changes got in.